installer/ClientInstaller.cs

```diff
@@ -172,7 +172,7 @@ class BuildInstaller
 		bool isAllLangs = false;
 
 		for (int i = 0; i < args.Length; i++) {
-			if (args[i] == "--version" && Regex.IsMatch(args[i + 1], @"^[0-9]\.[0-9]\.[0-9]\.[0-9]$")) {
+			if (args[i] == "--version" && Regex.IsMatch(args[i + 1], @"^([0-9]+\.){3}[0-9]+$")) {
```

Comment:
Did you verify that this syntax is actually supported by WixSharp/C#?

I still think that it'd be better to be explicit here and simply list each version part individually.

Reply:
@Kissaki What do you think?

Reply:
Well _did_ you verify that this would work in the first place? :eyes:

Reply:
I only tested the regex expression, however CI succeeded.

Reply:
Does the PR CI even build an installer? :eyes:

Reply:
What do I think about what?

What version of C# and .NET does WixSharp use?

Reply:
> Does the PR CI even build an installer? :eyes:

Yes.

> What do I think about what?
>
> What version of C# and .NET does WixSharp use?

About explicitly specifying every part of the version in the regex.

Reply:
I’m fine with either as I can see positives on both.

Having the parts laid out is longer, but also more explicit. So I tend towards that option. As Krzmbrzl argued for it too I would go for it.